pha-admin/src/main/java/com/store/pharmacy/common/serviceImpl/MessageServiceImpl.java

```diff
@@ -17,12 +18,12 @@
     MessageRepository messageRepository;
 
     @Override
-    public MessageOutput getByMessageCode(String messageCode) {
-        Message message = messageRepository.findByMessageCodeAndEnabledTrue(messageCode);
-        if(message != null) {
-            MessageOutput outParam = new MessageOutput(message.getMessageCode(), message.getText());
-            return outParam;
+    public MessageOutput getByMessageCode(String messageCode, String lang) {
+        Message message = messageRepository.findByMessageCodeAndLangAndEnabledTrue(messageCode, lang);
+        if(message == null) {
+            throw new DataNotFoundException();
         }
-        return null;
+        MessageOutput outParam = new MessageOutput(message.getMessageCode(), message.getText());
```

Comment:
Immediately return this expression instead of assigning it to the temporary variable "outParam".